package.json

```diff
@@ -19,7 +19,9 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
+    "lint": "npm run prettier  && npm run tslint",
+    "tslint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
+    "prettier": "prettier --write \"{src,test}/**/*.ts\"",
```

Comment:
might be nice to eliminate prettier altogether.  Is it really needed?  Note tslint does a lot of automated rules corrections.

Reply:
prettier is not using the lint config, because prettier is just a hardcore parse and rewrite code thing. i am not particularly attached to it, but I like the extremeness of the approach :-)